src/node_messaging.cc

```diff
@@ -517,7 +520,14 @@ void Message::MemoryInfo(MemoryTracker* tracker) const {
   tracker->TrackField("transferables", transferables_);
 }
 
-MessagePortData::MessagePortData(MessagePort* owner) : owner_(owner) { }
+MessagePortData::MessagePortData(
+    MessagePort* owner,
+    const std::string& name)
+    : owner_(owner),
+      group_(SiblingGroup::Get(name)) {
```

Comment:
It's not obvious but `SiblingGroup::Get()` returns nothing is `name` is empty here, so it's actually doing the right thing here. If `name` is empty (which it will be if we're using a regular one-to-one `MessageChannel`, then `group_` will not be set. I can add a comment to that effect so it's clear.

Reply:
I did notice that, and my point was that it would still be a bit clearer if attaching the sibling group would be a separate step (as it is for regular one-to-one `MessageChannel`s) – but not feeling too strongly about this

Reply:
For now, I'll leave this as is but will make a note to revisit

Reply:
I’ve leave an explicit TODO comment, tbh.